services/payments/src/simcore_service_payments/models/db.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+from decimal import Decimal
+from typing import Any, ClassVar
+
+from models_library.api_schemas_webserver.wallets import PaymentID
+from models_library.emails import LowerCaseEmailStr
+from models_library.products import ProductName
+from models_library.users import UserID
+from models_library.wallets import WalletID
+from pydantic import BaseModel, HttpUrl
+from simcore_postgres_database.models.payments_transactions import (
+    PaymentTransactionState,
+)
+
+_EXAMPLE_AFTER_INIT = {
+    "payment_id": "12345",
+    "price_dollars": 10.99,
+    "osparc_credits": 5.0,
+    "product_name": "osparc-plus",
+    "user_id": 123,
+    "user_email": "user@example.com",
+    "wallet_id": 123,
+    "comment": "This is a test comment.",
+    "invoice_url": None,
+    "initiated_at": "2023-09-27T10:00:00",
+    "state": PaymentTransactionState.PENDING,
+}
+
+#
+# NOTE: this will be moved to the payments service
+# NOTE: with https://sqlmodel.tiangolo.com/ we would only define this once!
```

Comment:
is that note really useful? maybe create an issue if you want to got that route.

Reply:
was old